src-built-in/preloads/zoom.js

```diff
@@ -237,7 +237,21 @@ const runZoomHandler = () => {
 	insertPopUp();
 
 	// Update the zoom configuration.
-	FSBL.Clients.ConfigClient.getValue({ field: "finsemble.Window Manager.zoom" }, zoomConfigHandler);
+	FSBL.Clients.LauncherClient.getComponentDefaultConfig(FSBL.Clients.WindowClient.getWindowIdentifier().componentType, (err, componentConfig) => {
```

Comment:
Should we check the `err` object?

Reply:
I'm hoping that try catch will deal with it and it will move on to the default.